chart/kubeapps/templates/apprepository-jobs-cleanup.yaml

```diff
@@ -0,0 +1,47 @@
+# The cleanup job is necessary to remove "apprepositories" created
+# in the bootstrap job or in the web application
```

Comment:
I would make this comment simpler, e.g.: "Clean up the AppRepository resources used by this Kubeapps instance"